pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -649,7 +667,7 @@ func (configgen *ConfigGeneratorImpl) buildInboundClusterForPortOrUDS(pluginPara
 		instance.Service.Hostname, instance.Endpoint.ServicePort.Port)
 	localityLbEndpoints := buildInboundLocalityLbEndpoints(pluginParams.Bind, instance.Endpoint.Port)
 	localCluster := buildDefaultCluster(pluginParams.Env, clusterName, apiv2.Cluster_STATIC, localityLbEndpoints,
-		model.TrafficDirectionInbound, pluginParams.Node, nil)
+		model.TrafficDirectionInbound, pluginParams.Node, nil, false) // TODO GregHanson mTLSReady always false?
```

Comment:
@rshriram chime in? Inbound cluster is for server envoy -> server app, which the answer should be `yes`?